src/server/svr_func.c

```diff
@@ -6699,20 +6699,18 @@ svr_load_jobscript(job *pj)
 		log_err(-1, __func__, log_buffer);
 		return NULL;
 	}
-	script = strdup(jobscr.script);
-	pbs_db_cleanup_resultset(conn);
 
-	if (script == NULL) {
+	if (jobscr.script == NULL) {
```

Comment:
Earlier this check served to handle strdup failure. Now as you have removed strdup, does this error logging still make sense? We also have another error logging prior to this which is more generic.

Reply:
Better to handle strdup failure in libdb itself and pass a specific return code back.

Reply:
This one is not addressed!